tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -40,18 +40,30 @@
 DEFAULT_FILE_NAME = 'data_tfrecord'
 
 
-def _ExamplePartitionKey(record: tf.train.Example,
-                         split_config: example_gen_pb2.SplitConfig) -> bytes:
-  """Generates key for partition for tf.train.Example."""
+def _ExamplePartitionKey(
+  record: Union[tf.train.Example, tf.train.SequenceExample],
```

Comment:
let have a unit test for both sequenceExample in base_example_gen_executor_test.py (you need another mock to create pcollection of sequenceexample, just like the one we have for now for tf.examples) and ImportExampleGen tests, we need to make sure importExampleGen can import sequenceExamples

Reply:
got it, i added a unit test for testImportSequenceExample